.github/workflows/heroku-deploy-main.yml

```diff
@@ -0,0 +1,20 @@
+name: Deploy
```

Comment:
Is there a reason you've got this in addition to the other one?
This gates on a branch, but it doesn't rely on the tests running successfully, so we've ended up with two checks being performed separately and either failing will still lead to a deployment! Instead we probably want to add an `if` check in the other pipeline - see my comment above